### DISCUSSION TITLE
SpringCloudGateway Error: Can not do async finish for the span repeatedly

### DISCUSSION THREAD

**Discussion by zhifu-guanxi** [apache/skywalking#8579] [Q&A]
My project use SpringCloudGateway 3.0.2 and skywalking version is 8.6.0.
Most applications performed normally, but gateway application has error log.
I find others have similar problems: https://lizz6.blog.csdn.net/article/details/109029308.
how I deal with it or ignore it?
![image](https://user-images.githubusercontent.com/42106399/155247102-7538b11a-a1ba-4e3e-aca3-75d5461655a4.png)

**Comment by wu-sheng** [comment]
You need to locate why. AFAIK, this seems an issue, but we don't know how you use it. 
Also, try the latest version and try to search existing issues and discussions. I remember some ppl discussed this before.
